kafka/consumer/main.go

```diff
@@ -28,14 +28,26 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs"
+	"github.com/microsoft/confidential-container-demos/kafka/util"
 )
 
 var keyEnabled bool
 
 const eventHubNamespace = "EVENTHUB_NAMESPACE"
 const eventHub = "EVENTHUB"
 
+var logLocation = util.GetEnv("LOG_FILE")
+
 func main() {
+
+	if len(logLocation) > 0 {
+		f, err := os.OpenFile(logLocation, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0740)
+		if err != nil {
+			log.Printf("Unable to open file log location: %s", err.Error())
```

Comment:
Did this mean to exit or continue?

Reply:
Continue. We don't have to stop the execution because log is not available. Also this is a copy and paste from the log code you addded a while back.